api/src/main/java/io/grpc/TimeUtils.java

```diff
@@ -19,7 +19,7 @@
 import java.time.Duration;
 
 public final class TimeUtils {
-  public static long convert(Duration duration) {
+  public static long convertToNanos(Duration duration) {
     try {
```

Comment:
@ejona86 Why is Codecov complaining even though the class is in fact covered by unit tests?

Reply:
@SreeramdasLavanya can you add the a private constructor?